### ISSUE TITLE
Bug: with update query

### ISSUE THREAD

**Issue by volodymyr-hordiienko** [hyperf/database-pgsql-incubator#2]
Looks like invalid postgres query created when Model->save() (pdo_pgsql). Think there should not to be prefix in set section. 

```
[ERROR] SQLSTATE[42703]: Undefined column: 7 ERROR:  column "users" of relation "users" does not exist
LINE 1: update "users" set "confirmed" = $1, "users"."updated_at" = ...
                                             ^ (SQL: update "users" set "confirmed" = 2022-06-20 14:48:17, "users"."updated_at" = 2022-06-20 14:48:17 where "id" = 1)[1082] in /opt/www/vendor/hyperf/database/src/Connection.php
[ERROR] #0 /opt/www/vendor/hyperf/database/src/Connection.php(1039): Hyperf\Database\Connection->runQueryCallback()
#1 /opt/www/vendor/hyperf/database/src/Connection.php(394): Hyperf\Database\Connection->run()
#2 /opt/www/vendor/hyperf/database/src/Connection.php(339): Hyperf\Database\Connection->affectingStatement()
#3 /opt/www/vendor/hyperf/db-connection/src/Connection.php(68): Hyperf\Database\Connection->update()
#4 /opt/www/vendor/hyperf/db-connection/src/Traits/DbConnection.php(56): Hyperf\DbConnection\Connection->__call()
#5 /opt/www/vendor/hyperf/database/src/Query/Builder.php(2277): Hyperf\DbConnection\Connection->update()
#6 /opt/www/vendor/hyperf/database/src/Model/Builder.php(805): Hyperf\Database\Query\Builder->update()
#7 /opt/www/vendor/hyperf/database/src/Model/Model.php(1355): Hyperf\Database\Model\Builder->update()
#8 /opt/www/vendor/hyperf/database/src/Model/Model.php(575): Hyperf\Database\Model\Model->performUpdate()
#9 /opt/www/app/Repository/AbstractRepository.php(30): Hyperf\Database\Model\Model->save()
#10 /opt/www/runtime/container/proxy/App_Service_Auth_ConfirmUserService.proxy.php(35): App\Repository\AbstractRepository->save()
```

**Comment by zhuangyuanhui** [comment]
I see. Thank you!
I will fix it in the near future
You can add Issue



---- Replied Message ----
| From | Vladimir ***@***.***> |
| Date | 06/20/2022 22:52 |
| To | ***@***.***> |
| Cc | ***@***.***> |
| Subject | [hyperf/database-pgsql-incubator] Bug with update query (Issue #2) |

Looks like invalid postgres query created when Model->save() (pdo_pgsql). Think there should not to be prefix in set section.

[ERROR] SQLSTATE[42703]: Undefined column: 7 ERROR:  column "users" of relation "users" does not exist
LINE 1: update "users" set "confirmed" = $1, "users"."updated_at" = ...
                                             ^ (SQL: update "users" set "confirmed" = 2022-06-20 14:48:17, "users"."updated_at" = 2022-06-20 14:48:17 where "id" = 1)[1082] in /opt/www/vendor/hyperf/database/src/Connection.php
[ERROR] #0 /opt/www/vendor/hyperf/database/src/Connection.php(1039): Hyperf\Database\Connection->runQueryCallback()
#1 /opt/www/vendor/hyperf/database/src/Connection.php(394): Hyperf\Database\Connection->run()
#2 /opt/www/vendor/hyperf/database/src/Connection.php(339): Hyperf\Database\Connection->affectingStatement()
#3 /opt/www/vendor/hyperf/db-connection/src/Connection.php(68): Hyperf\Database\Connection->update()
#4 /opt/www/vendor/hyperf/db-connection/src/Traits/DbConnection.php(56): Hyperf\DbConnection\Connection->__call()
#5 /opt/www/vendor/hyperf/database/src/Query/Builder.php(2277): Hyperf\DbConnection\Connection->update()
#6 /opt/www/vendor/hyperf/database/src/Model/Builder.php(805): Hyperf\Database\Query\Builder->update()
#7 /opt/www/vendor/hyperf/database/src/Model/Model.php(1355): Hyperf\Database\Model\Builder->update()
#8 /opt/www/vendor/hyperf/database/src/Model/Model.php(575): Hyperf\Database\Model\Model->performUpdate()
#9 /opt/www/app/Repository/AbstractRepository.php(30): Hyperf\Database\Model\Model->save()
#10 /opt/www/runtime/container/proxy/App_Service_Auth_ConfirmUserService.proxy.php(35): App\Repository\AbstractRepository->save()

**Comment by limingxinleo** [comment]
fixed in v0.1.2
